File: java/fibunacci/src/com/company/fibunacciMath.java
package com.company;

/**
 * Created by mayelespino on 6/24/17.
 */

public class fibunacciMath {
    public Double calculate(int number) {
        Double root5 = Math.sqrt(5.0);
        Double gr = (1 + root5) / 2.0;
        Double igr = 1 - gr;
        Double value = (Math.pow(gr, number) - Math.pow(igr, number)) / root5;
        //return(Math.round(value));
        return(value);
    }
}

File: python/fibonacci/fibunacci-math.py
#!/usr/bin/python
# Got this solution from http://www.nerdparadise.com/tech/interview/fibonacci/
from math import sqrt

def fibonacci(n):
    root5 = sqrt(5)
    gr = (1 + root5) / 2
    igr = 1 - gr
    value = (pow(gr, n) - pow(igr, n)) / root5

    # round it to the closest integer since floating 
    # point arithmetic cannot be trusted to give
    # perfect integer answers. 
    return round(value,9)

target = 100
print(target)
print (fibonacci(target))
# try:
#     aNumber=int(raw_input('Give me a number:'))
# except ValueError:
#     print ("Not a number"

#print fibonacci(aNumber)